calicoctl/commands/resources.go

```diff
@@ -101,31 +101,32 @@ func getResourceFromArguments(args map[string]interface{}) (unversioned.Resource
 	orchestrator := argutils.ArgStringOrBlank(args, "--orchestrator")
 	resScope := argutils.ArgStringOrBlank(args, "--scope")
 	switch strings.ToLower(kind) {
-	case "node", "nodes":
+	case "node", "nodes", "no", "nos":
 		p := api.NewNode()
 		p.Metadata.Name = name
 		return *p, nil
-	case "hostendpoint", "hostendpoints":
+	case "hostendpoint", "hostendpoints","hostep", "hosteps", "hep", "heps":
 		h := api.NewHostEndpoint()
 		h.Metadata.Name = name
 		h.Metadata.Node = node
 		return *h, nil
-	case "workloadendpoint", "workloadendpoints":
+	case "workloadendpoint", "workloadendpoints", "workloadep", "workloadeps",
```

Comment:
Personally I don't see the value in the only partially abbreviated versions (workloadep, workloadeps).

I'd be happy with `wep` and `weps`, but that's just me.